plugins/generator-1.18.2/forge-1.18.2/procedures/loot_table_foreach.java.ftl

```diff
@@ -0,0 +1,15 @@
+<#include "mcelements.ftl">
+if (${input$entity} instanceof LivingEntity _entity && !_entity.level.isClientSide() && _entity.getServer() != null) {
+	try {

```

Comment:
Why is this try? Are there known exceptions and is there not a better way to handle them if the answer is yes?

Reply:
This handles exceptions that _may be thrown_ by `ObfuscationReflectionHelper` (similar to IDEA not letting you run the program unless you catch all the exceptions that _may be_ thrown by your code).

Reply:
I Thought ObfuscationReflectionHelper handled exceptions on its own, my mistake.

> similar to IDEA not letting you run the program unless you catch all the exceptions that may be thrown by your code

I thought we were over with this, but once again this sounds very passive-aggressive. It can probably be assumed the maintainers of this project understand the concept of try-catch in Java, don't you think? I really can't understand why we keep coming back to this, and really don't want to do this again.

Reply:
I intended this to be a comparison (A here works just like our old good B), I didn't even think it would sound like that bad/aggresive when writing the comment. Terribly sorry 😖